backend/agents/create_tasks_agent.py

```diff
@@ -1,41 +1,28 @@
+import json
+
+from prompts.prompting import PromptEngine
 from utils import Config
 from utils import call_model
 import logging
 
 logger = logging.getLogger(__name__)
 config = Config()
 
-create_tasks_prompt = """
-    You are an agent who specialises in breaking down questions into smaller tasks that are performed to complete
-    the question.
-
-    You are dedicated to performing the job of breaking down a user question into up to 5 tasks that are easy
-    to understand and manageable in size.
-
-    A user will provide a goal that they want to perform and you will create tasks that when all of them are performed,
-    will provide an answer for the user question.
-
-    To determine the tasks you will create, you will use reasoning, any context you have and will attempt to criticise
-    each task in order to find the best way possible for tasks to answer the overall question.
+def create_tasks(user_prompt: str) -> str:
 
-    If you are unable to breakdown the question into tasks, say that unfortunately you are unable to break down
-    the question.
+    # TODO: Make single source of agent choice knowledge
+    agents = "unresolvable_task, database_agent, fiancial_advisor_agent, web_search_agent"
```

Comment:
This contains an agent called `unresolvable_task`. The `create-tasks.j2` prompt template also appends another agent called `unresolvable_agent`.

Are these intentionally different agents?

Reply:
No 🤦 fixed